tests/src/Unit/Service/Context/PageContextTest.php

```diff
@@ -83,67 +83,126 @@ class PageContextTest extends UnitTestCase {
    */
   private $titleResolver;
 
+  /**
+   * Language Manager.
+   *
+   * @var Drupal\Core\Language\LanguageManagerInterface|\PHPUnit_Framework_MockObject_MockObject
+   */
+  private $languageManager;
+
+  /**
+   * Language Object.
+   * 
+   * @var Drupal\Core\Language\Language|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
Replace all `Drupal\Core\Language\Language` by use of `Drupal\Core\Language\LanguageInterface`.

Reply:
Replaced all instances of ```Drupal\Core\Language\Language``` with ```Drupal\Core\Language\LanguageInterface```